docs/source/reference/numpysupported.rst

```diff
@@ -582,6 +582,46 @@ Modules
 ``random``
 ----------
 
+Generator Objects
+''''''''''''
+Numba supports `numpy.random.Generator()` objects and their respective
+methods. Numpy's `Generator` objects rely on `BitGenerator` to manage state
+and generate the random bits, which are then transformed into random
+values from useful distributions. Numba `unbox`es the `Generator` objects
+and maintians a reference to the underlying `BitGenerator` objects using Numpy's
+`ctypes` interface bindings. Hence `Generator` objects can cross the JIT boundary
+and their functions be used within Numba-Jit code. Note that since only references
+to `BitGenerator` objects are maintained, any change to the state of a particular
+`Generator` object outside Numba code would affect the state of `Generator`
+inside the Numba code.
+
```

Comment:
Look at the build at:

https://numba--8031.org.readthedocs.build/en/8031/reference/numpysupported.html

You will need the following:

```suggestion
.. code-block:: python

```

To render the python example correctly. Otherwise it will look like:

<img width="1118" alt="Screen Shot 2022-05-16 at 10 28 47" src="https://user-images.githubusercontent.com/41563/168551850-6db7d10f-0292-43a9-acb5-b63b1684e650.png">